src/types/user.ts

```diff
@@ -516,6 +516,23 @@ export default /* GraphQL */ `
     id: ID!
     address: String!
     createdAt: DateTime!
+
+    """NFT assets owned by this wallet address"""
+    assets: [NFTAsset!]!
```

Comment:
* `nfts` instead of `assets`?
* `[NFTAsset!]` instead of `[NFTAsset!]!`?

Reply:
it can always return a `[]` instead of null, might be helpful for frontend sometimes